src/features/tags/hooks/useEditTag.ts

```diff
@@ -0,0 +1,65 @@
+import { EditTag } from '../components/TagManager/types';
+import useCreateTagGroup from './useCreateTagGroup';
+import { tagUpdate, tagUpdated } from '../store';
+import { useApiClient, useAppDispatch } from 'core/hooks';
+import { ZetkinTag, ZetkinTagPatchBody } from 'utils/types/zetkin';
+
+export default function useEditTag(
```

Comment:
As hinted in my other comment, I think this should be a `useTagMutations()` hook and that it should return an object containing an `updateTag()` function that is similar to the single function returned by this hook right now.

I do not think it should deal in callbacks. Instead, that type of logic can be solved by awaiting the `updateTag()` function and using it's return value.